backend/src/App.ts

```diff
@@ -1,10 +1,62 @@
 import express from 'express';
+import {createConnection, getRepository, Repository} from "typeorm";
+import bodyParser from "body-parser";
+import {User} from "./entity/User";
+import {Promotion} from "./entity/Promotion";
+import {Discount} from "./entity/Discount";
+import {promotions_sample, saved_promotions_mapping, users_sample} from "./resources/Data";
 
-const app = express();
-const PORT = 8000;
+//todo: ormconfig.json should not have synchronize and drop schema as true for production
+createConnection()
+    .then(async connection => {
+        const app = express();
+        app.use(bodyParser.json())
+        const PORT = 8000;
 
-app.get('/', (req, res) => res.send('Hello World'));
+        // todo: create custom repositories
+        let userRepository: Repository<User> = getRepository(User);
+        let promotionRepository: Repository<Promotion> = getRepository(Promotion);
+        let discountRepository: Repository<Discount> = getRepository(Discount);
 
-app.listen(PORT, () => {
-    console.log(`⚡️[server]: Server is running at http://localhost:${PORT}`);
-});
\ No newline at end of file
+        // persist entities into database
+        for (let user of users_sample) {
+            await userRepository.save(user);
+        }
+
+        for (let promotion of promotions_sample) {
+            await promotionRepository.save(promotion)
+        }
+
+        for (let [user, promotions] of saved_promotions_mapping) {
+            user.savedPromotions = promotions;
+            await userRepository.save(user);
+        }
```

Comment:
We can either use this to load the data, or run psql command, see the read.me for backend
https://github.com/ubclaunchpad/foodies/blob/backend-orm/backend/README.md#to-import-data-from-init_datasql-run-this-command-inside-a-command-prompt-make-sure-the-schema-is-setup-and-it-is-currently-empty

![image](https://user-images.githubusercontent.com/49849754/98207772-05e86500-1ef1-11eb-9639-c7fa9f246e63.png)

In this PR, there are two ways we can load data, either through 
1. https://github.com/ubclaunchpad/foodies/blob/backend-orm/backend/src/resources/Data.ts
2. https://github.com/ubclaunchpad/foodies/blob/backend-orm/backend/src/resources/init_data.sql
  - credits to @yunishimura0716 for the sample data